### PR TITLE
Add counter tests

### PR DESCRIPTION
**Pull request by kmatsuleva**

### CODE REVIEW

**Comment by mgenov** [comment]
PR needs to be rebased with the upstream.

File: counter.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const counter = (state, action) => {
```
Comment (mgenov) [inline counter.js]: You can use default values of ES6. 

const counter = (state = 0, action) will removed the need of the if statement.

File: counter.test.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+const { counter } = require('./counter');
+
+
+describe("With state ", () => {
```
Comment (mgenov) [inline counter.test.js]: Minor naming improvement: 
```jsx
describe('Counter is', () => {
  test('incrementing from 0 to 1', () => {
  
   })
})
```

**Comment by kmatsuleva** [comment]
@mgenov  , added default value to state, changed test suites description, fixed merge conflict with master

**Pull request merged**